gateway/src/main/java/io/zeebe/gateway/EndpointManager.java

```diff
@@ -400,32 +392,36 @@ public static StatusRuntimeException convertThrowable(final Throwable cause) {
 
     if (cause instanceof BrokerErrorException) {
       status = mapBrokerErrorToStatus(((BrokerErrorException) cause).getError());
+      // When there is back pressure, there will be a lot of `RESOURCE_EXHAUSTED` errors and the log
+      // can get flooded. Until we find a way to limit the number of log messages,
+      // let's do not log them.
+      if (status.getCode() != Status.RESOURCE_EXHAUSTED.getCode()) {
+        Loggers.GATEWAY_LOGGER.error(GRPC_ERROR_TEMPLATE + "received error from broker", cause);
```

Comment:
Nitpick: not sure we gain much by "dry"ing the use of GRPC_ERROR_TEMPLATE, I'd personally just put each error reasons as a separate static string, e.g.

```java
private static final GRPC_BROKER_ERROR_MESSAGE = "Expected to handle gRPC request, but the broker returned an error";
private static final GRPC_BROKER_REJECTION_MESSAGE = "Broker rejected request";
private static final GRPC_TIME_OUT_MESSAGE = "Request timed out between gateway and broker";
// ...
```

(just a nit, so you can ignore this tbh - also reworded some messages since they aren't error messages)

Reply:
Not sure what you meant by message rewording. I've unDRY-ed the log messages, but do you want anything else changed here? I wasn't sure.